src/engine/strat_engine/thinpool/mdv.rs

```diff
@@ -236,6 +240,11 @@ impl MetadataVol {
         self.dev.set_table(get_dm(), table)?;
         Ok(())
     }
+
+    /// The maximum number of filesystems that can be recorded in the MDV.
+    pub fn max_fs_limit(&self) -> u64 {
```

Comment:
I believe this is on overprediction, because the data size of the xfs filesystem on the mdv will be less than the logical size of the thin device.

Reply:
What I'm ultimately calculating is the number of files that can be held in the single MDV filesystem which I believe @drckeefe said was the total size of the filesystem divided by the block size. Did I misunderstand this?

Reply:
I believe that what you've got there is the size of the linear device the filesystem is on. The amount of data that can be written will be less, due to xfs metadata, but also potential layout effects.

Reply:
Okay I think this is resolved!